test/e2e/azure_test.go

```diff
@@ -460,9 +460,9 @@ var _ = Describe("Workload cluster creation", func() {
 				},
 			}, result)
 
-			Context("Validating AKS Resources", func() {
-				AKSResourcesValidationSpec(ctx, func() AKSResourcesValidationSpecInput {
-					return AKSResourcesValidationSpecInput{
+			Context("Validating AKS time synchronization", func() {
+				AzureDaemonsetTimeSyncSpec(ctx, func() AzureTimeSyncSpecInput {
```

Comment:
So is this replacing the check we were doing before? If so, is that code still being used anywhere (looks like no) / should we remove it?

Reply:
ah good catch. Thought I got it in my patch when I split the diff from the other PR.